monorepo-migration/sync-focus.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+set -ex
+
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+CURRENT_REPO_PATH="$(dirname -- "$SCRIPT_DIR")"
+
+REPO_NAME_TO_SYNC='focus-ios'
+MAIN_BRANCH_NAME='main'
+
+CURRENT_BETA_VERSION=$(grep -A 1 'BITRISE_BETA_VERSION' bitrise.yml | awk '/-/{getline; print}' | tr -d "'")
```

Comment:
Some changes in this line to make it working: 
`CURRENT_BETA_VERSION=$(grep -A 1 'BITRISE_BETA_VERSION:' bitrise.yml | awk -F ':' '{print $NF}' | tr -d "'" | tr -d ' ' | tr -d '.0')`